### PR TITLE
[zh-cn] add 2024-08-21-cgroup-driver-from-cri.md chinese version

### PR DESCRIPTION
**Pull request by zhangxyjlu**
fixed: #48297

Translate Chinese version for blog：2024-08-21-cgroup-driver-from-cri.md

Preview：https://deploy-preview-48300--kubernetes-io-main-staging.netlify.app/zh-cn/blog/2024/08/21/cri-cgroup-driver-lookup-now-beta/

### CODE REVIEW

File: content/zh-cn/blog/_posts/2024-08-21-cgroup-driver-from-cri.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+Kubernetes clusters. On Linux systems, there are two different cgroup drivers:
+`cgroupfs` and `systemd`. In the past, both the [kubelet](/docs/reference/command-line-tools-reference/kubelet/)
+and CRI implementation (like CRI-O or containerd) needed to be configured to use
+the same cgroup driver, or else the kubelet would exit with an error. This was a
+source of headaches for many cluster admins. However, there is light at the end of the tunnel!
+-->
+一直以来，为新运行的 Kubernetes 集群配置正确的 cgroup 驱动程序是用户的一个痛点。
+在 Linux 系统中，存在两种不同的 cgroup 驱动程序：`cgroupfs` 和 `systemd`。
+过去，[kubelet](/zh-cn/docs/reference/command-line-tools-reference/kubelet/) 和 CRI
+实现（如 CRI-O 或 containerd）都需要配置为使用相同的 cgroup 驱动程序， 否则 kubelet 会报错并退出。
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2024-08-21-cgroup-driver-from-cri.md:31]:
```suggestion
实现（如 CRI-O 或 containerd）需要配置为使用相同的 cgroup 驱动程序， 否则 kubelet 会报错并退出。
```

File: content/zh-cn/blog/_posts/2024-08-21-cgroup-driver-from-cri.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+In v1.28.0, the SIG Node community introduced the feature gate
+`KubeletCgroupDriverFromCRI`, which instructs the kubelet to ask the CRI
+implementation which cgroup driver to use. A few minor releases of Kubernetes
+happened whilst we waited for support to land in the major two CRI implementations
+(containerd and CRI-O), but as of v1.31.0, this feature is now beta!
+-->
+## 自动检测 cgroup 驱动程序
+
+在 v1.28.0 版本中，SIG Node 社区引入了 `KubeletCgroupDriverFromCRI` 特性门控，
+它指示 kubelet 向 CRI 实现请求使用哪个 cgroup 驱动程序。在主要的两个 CRI 实现（containerd
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2024-08-21-cgroup-driver-from-cri.md:46]:
```suggestion
它指示 kubelet 向 CRI 实现询问使用哪个 cgroup 驱动程序。在两个主要的 CRI 实现（containerd
```

File: content/zh-cn/blog/_posts/2024-08-21-cgroup-driver-from-cri.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+和 CRI-O）支持该功能的过程中，Kubernetes 经历了几次小版本发布，但从 v1.31.0 版本开始，此功能现已进入 beta 阶段！
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2024-08-21-cgroup-driver-from-cri.md:47]:
```suggestion
和 CRI-O）增加对该功能的支持这段期间，Kubernetes 经历了几次小版本发布，但从 v1.31.0 版本开始，此功能现已进入 beta 阶段！
```

File: content/zh-cn/blog/_posts/2024-08-21-cgroup-driver-from-cri.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+除了设置特性门控之外，集群管理员还需要确保 CRI 实现版本足够新：
+
+- containerd：v2.0.0 版本开始支持
+- CRI-O：v1.28.0 版本开始支持
+
+<!--
+Then, they should ensure their CRI implementation is configured to the
+cgroup_driver they would like to use.
+-->
+然后，他们应该确保其 CRI 实现已配置为他们想要使用的 cgroup 驱动程序。
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2024-08-21-cgroup-driver-from-cri.md:65]:
```suggestion
然后，他们应该确保配置其 CRI 实现使用他们想要的 cgroup 驱动程序。
```

File: content/zh-cn/blog/_posts/2024-08-21-cgroup-driver-from-cri.md [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+<!--
+## Future work
+
+Eventually, support for the kubelet's `cgroupDriver` configuration field will be
+dropped, and the kubelet will fail to start if the CRI implementation isn't new
+enough to have support for this feature.
+-->
+## 未来工作
+
+最终，kubelet 的 `cgroupDriver` 配置字段的支持将会被移除，如果 CRI 实现的版本不够新，无法支持该功能，kubelet 将无法启动。
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2024-08-21-cgroup-driver-from-cri.md:76]:
```suggestion
最终，kubelet 对 `cgroupDriver` 配置字段的支持将会被移除，如果 CRI 实现的版本不够新，无法支持此功能，kubelet 将无法启动。
```

**Comment by windsonsea** [comment]
/lgtm

Note: You can mark those comments as **Resolved** after pushing your change.

**Pull request merged**